modules/openid_auth/MIT-LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright 2018 Michael Bastos
```

Comment:
Copy CC public domain LICENSE from one of the other modules, as work done on behalf of the government this goes in the public domain.